jwql/website/apps/jwql/urls.py

```diff
@@ -53,6 +53,7 @@
     path('', views.home, name='home'),
     path('about/', views.about, name='about'),
     path('dashboard/', views.dashboard, name='dashboard'),
+    path('edb/', views.engineering_database, name='edb'),
```

Comment:
Please add your name (and Matt's, if you don't mind) to the author list for this page!